### ISSUE TITLE
Using this setup for BLFS

### ISSUE THREAD

**Issue by MaxLeiter** [emmett1/lfs-scripts#6]
Hi,
I'm interested in a similar setup for BLFS (with custom patches and stuff applied). Would CRUX pkgutils + this repo setup be a good option? Seems like it to me; I just need to either crawl or manually swap directions + packages out with the BLFS ones.

**Comment by emmett1** [comment]
Yes, you can use this script to build B/LFS + pkgutils. But its been a while i'm update it. You can update it first before running the script.
